src/app/csv-collection/csv-collection-detail/csv-collection-detail.component.html

```diff
@@ -103,86 +83,58 @@ <h2 class="top10">Transfers</h2>
-    <tr *ngFor="let file of flowHolder.files | slice:0:10; let index=index">
-      <td class="vert-align">{{index + 1}}</td>
-      <td class="vert-align">{{file.name}}</td>
-      <td class="vert-align">{{file.size | bytes}}</td>
-      <td *ngIf='!file.error'>
-        <p-progressBar aria-label="Upload progress" showValue='false' [value]="file.progress()*100.0" />
-      </td>
-      <td>
-        <span *ngIf='file.error'>{{file.errorMesage|| "Unknown upload error."}}</span>
-      </td>
-    </tr>
+      <tr *ngFor="let file of flowHolder.files | slice:0:10; let index=index">
+        <td class="vert-align">{{index + 1}}</td>
+        <td class="vert-align">{{file.name}}</td>
+        <td class="vert-align">{{file.size | bytes}}</td>
+        <td *ngIf='!file.error'>
+          <p-progressBar aria-label="Upload progress" showValue='false' [value]="file.progress()*100.0" />
+        </td>
+        <td>
+          <span *ngIf='file.error'>{{file.errorMesage|| "Unknown upload error."}}</span>
+        </td>
+      </tr>
     </tbody>
   </table>
 </div>
 
-<h2 style="margin-top: 2%">CSV Collection files</h2>
+<h2 style="margin-top: 2%">CSV Collection files ({{csvCollection.csvTotalSize | bytes}})</h2>
+
 <div class="card">
-  <div class='flex flex-column md:flex-row md:justify-content-between py-2'>
-    <div class='mb-4 md:mb-0 vertical-align-baseline'>
-      Total size: {{csvCollection.csvTotalSize | bytes}}
-    </div>
-    <div class='mb-3 md:mb-0' style="position:relative">
-    </div>
-    <div class="mb-3 md:mb-0 vertical-align-middle">
-    </div>
-    <div class='mb-2 md:mb-0'>
-      <p-button icon="pi pi-trash" *ngIf="canEdit() && !csvCollection.locked && !csvCollection.sourceJob"
-                size="small"
-                severity="danger"
-                pTooltip="Delete all files"
-                placement="left"
-                (click)="deleteAllCsvFiles()">
-      </p-button>
-    </div>
-  </div>
-  <p-table [value]="csv" [lazy]="true" *ngIf="csvCollection.id"
-           (onLazyLoad)="getCsvFiles($event)"
-           dataKey="id"
-           [tableStyle]="{ 'min-width': '75rem' }"
-           [paginator]="true"
-           [rows]="10"
-           [rowsPerPageOptions]="[10, 25, 50]"
-           [totalRecords]="resultsLengthCsv">
+  <p-table [value]="csv" [tableStyle]="{ 'min-width': '50rem' }">
     <ng-template pTemplate="header">
-      <tr>
-        <th pSortableColumn="fileName">
-          Name <p-sortIcon field="name" />
-        </th>
-        <th pSortableColumn="fileSize">
-          Size <p-sortIcon field="fileSize" />
-        </th>
-        <th>Actions</th>
-      </tr>
+        <tr>
+            <th>Name</th>
+            <th>Size</th>
+            <th>Actions</th>
+        </tr>
     </ng-template>
-    <ng-template pTemplate="body" let-csvFile>
-      <tr>
-        <td>
-          {{ csvFile.fileName }}
-          <span [hidden]="!csvFile.importError"><br>{{csvFile.importError}}</span>
```

Comment:
I don't see the `importError` display in the changes

Reply:
Done